geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-common/src/main/scala/org/locationtech/geomesa/fs/storage/common/package.scala

```diff
@@ -23,6 +24,7 @@ package object common {
 
   val RenderOptions: ConfigRenderOptions = ConfigRenderOptions.concise().setFormatted(true)
   val ParseOptions: ConfigParseOptions = ConfigParseOptions.defaults()
+  val fileValidationEnabled: SystemProperty = SystemProperty("geomesa.fs.validate.file", "false")
```

Comment:
```suggestion
  val FileValidationEnabled: SystemProperty = SystemProperty("geomesa.fs.validate.file", "false")
```
convention in scala is to upper-case constants (i.e. vals inside objects)